src/Shared/SFA.DAS.SharedOuterApi/InnerApi/Requests/Roatp/GetAllCourseTypesRequest.cs

```diff
@@ -0,0 +1,7 @@
+﻿using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.SharedOuterApi.InnerApi.Requests.Roatp;
+public class GetAllCourseTypesRequest : IGetApiRequest
```

Comment:
put coverage on this.  In this case, it's literally just check the request.GetUrl is 'course-types'